content/blog/basic-commands.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 3 basic commands to teach your dog
+description: These commands are essential for any dog owner to know.
+date: '2022-04-24T23:48:55.749Z'
+lastmod: '2022-04-24T01:42:32.610Z'
+draft: false
+tags: []
+categories: []
+type: blog
+preview: /uploads/looking_dog.jpg
+---
+
+Some new dog owners think that they don't need to start teaching their dogs any commands until they reach a certain age. It's actually very important to start teaching your dog certain actions, or commands, as early as possible. Starting your training at an early age can help to build a solid foundation between you and your dog. 
+
+There are several basic commands that every dog owner should teach their dog. These commands can help you manage some common behavioral problems, like growling or trying to slip off their leash. 
+
+## Coming to you
+
+Sometimes your dog may slip off his leash if he sees a squirrel running by or escape into the backyard while you're taking out the trash. In these instances, it's best to use the come command to direct your dog towards you. If you plan on taking your dog off his leash, this is an essential command to know. 
```

Comment:
Suggestion/ question: Should we call attention to dog commands by highlighting them by bolding or italicizing them? Such as, "...use the _come_ command to direct your do towards you."